ansible/README.md

```diff
@@ -141,6 +142,22 @@ elasticsearch_network_interface=ens3
 poetry run ansible-playbook -i hosts.ini elasticsearch.yml -vv
 ```
 
+### Minio
+
+* In your 'hosts.ini' file, in the `[minio:vars]` section, set 'minio_network_interface'. For example:
```

Comment:
This should state that specifying a network interface is mandatory (the defaults don't work correctly).